pkg/cmd/cli/cmd/deploy.go

```diff
@@ -289,9 +289,9 @@ func (c *retryDeploymentCommand) retry(config *deployapi.DeploymentConfig, out i
 	if status := deployutil.DeploymentStatusFor(deployment); status != deployapi.DeploymentStatusFailed {
 		message := fmt.Sprintf("#%d is %s; only failed deployments can be retried.\n", config.LatestVersion, status)
 		if status == deployapi.DeploymentStatusComplete {
-			message += fmt.Sprintf("You can start a new deployment using the --latest option.")
+			message += "You can start a new deployment using the --latest option."
 		} else {
-			message += fmt.Sprintf("Optionally, you can cancel this deployment using the --cancel option.", config.LatestVersion)
+			message += fmt.Sprint("Optionally, you can cancel this deployment using the --cancel option.", config.LatestVersion)
```

Comment:
I wasn't sure if the intent here was to concatenate that `LatestVersion` at the end for some reason.